BuildResidentialHPXML/measure.rb

```diff
@@ -5061,6 +5073,28 @@ def self.set_pv_systems(hpxml, runner, args, epw_file)
     end
   end
 
+  def self.set_battery(hpxml, runner, args)
+    return if args[:battery_location] == 'none'
+
+    if args[:battery_location] != Constants.Auto
+      location = args[:battery_location]
+    end
+
+    if args[:battery_power] != Constants.Auto
+      rated_power_output = Float(args[:battery_power])
+    end
+
+    if args[:battery_capacity] != Constants.Auto
+      nominal_capacity = Float(args[:battery_capacity])
+    end
+
+    hpxml.batteries.add(id: "Battery#{hpxml.batteries.size + 1}",
+                        type: HPXML::BatteryTypeLithiumIon,
+                        location: location,
+                        rated_power_output: rated_power_output,
```

Comment:
You're asking for power in kW but HPXML wants it in W. That needs to be fixed.

Reply:
Please audit everywhere that `rated_power_output` is used to make sure you're using the correct units.

Reply:
Should we ask for it in kW or W?

Reply:
I'd probably use W to be consistent with HPXML itself.